experiments/interpretation/dutchf3_patch/local/azureml_requirements.txt

```diff
@@ -0,0 +1,13 @@
+git+https://github.com/olgaliak/seismic-deeplearning.git@azureml-train-pipeline#egg=cv_lib&subdirectory=cv_lib
```

Comment:
please pull from /microsoft. Also is there a way to specify master branch here? We're on staging by default which might break one of these days

Reply:
Unfortunately doing that would require the __init__.py file that I added so this wouldn't work until that file is added to /microsoft and /master (there is a way to specify the master branch). I have a note about changing it to microsoft/staging before the merge in the PR comment. Just leaving for now so it can be tested